src/main/scala/is/hail/io/annotators/BedAnnotator.scala

```diff
@@ -59,7 +63,18 @@ object BedAnnotator {
           // transform BED 0-based coordinates to Hail/VCF 1-based coordinates
           val start = spl(1).toInt + 1
           val end = spl(2).toInt + 1
-          val interval = Interval(Locus(chrom, start), Locus(chrom, end), true, false)
+
+          val startLocus = gr match {
```

Comment:
would it be useful to capture this in a function like you've done with `TLocus.schemaFromGR`? You seem to use this in quite a few places. Maybe on the `Locus` object or something:

```
def fromGR(contig: String, position: Int, gr: Option[GenomeReference]): Annotation =
    gr match {
        case Some(ref) => Locus(contig, position, ref)
        case None => Annotation(contig, position)
    }
```